File: src/nomanssave/ae.java
package nomanssave;

class ae extends G {
   // $FF: synthetic field
   final X bV;

   ae(X var1) {
      this.bV = var1;
   }

   protected String g(String var1) {
      gj var2 = (gj)X.k(this.bV).getSelectedItem();
      if (var2 == null) {
         return "";
      } else {
         var1 = var1.trim();
         if (!var1.equals(var2.cN())) {
            var2.ab(var1);
            X.e(this.bV).setText(var1);
         }

         return var1;
      }
   }
}

File: NMSSaveEditor/nomanssave/lower/ae.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{
class ae : G {
   X bV;

   ae(X var1) {
      this.bV = var1;
   }

   protected string g(string var1) {
      gj var2 = (gj)X.k(this.bV).SelectedItem;
      if (var2 == null) {
         return "";
      } else {
         var1 = var1.Trim();
         if (!var1.Equals(var2.cN())) {
            var2.ab(var1);
            X.e(this.bV).Text = (var1);
         }

         return var1;
      }
   }
}

}
